src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/Controllers/OrganisationControllerTests/WhenIConfirmAddOfOrganisation.cs

```diff
@@ -98,4 +99,63 @@ public async Task ThenIAmSuppliedTheHashedAgreementIdForANewSearch()
             Assert.AreEqual(testHashedAgreementId, result.RouteValues["HashedAgreementId"]);
         }
     }
+    public class WhenIConfirmAddOfOrganisationAndIAmNotTheOwner
+    {
+        private OrganisationController _controller;
+        private Mock<OrganisationOrchestrator> _orchestrator;
+        private Mock<IAuthenticationService> _owinWrapper;
+        private Mock<IAuthorizationService> _featureToggle;
+        private Mock<IMultiVariantTestingService> _userViewTestingService;
+        private Mock<IMapper> _mapper;
+        private Mock<ILog> _logger;
+        private Mock<ICookieStorageService<FlashMessageViewModel>> _flashMessage;
+
+        private const string testHashedAgreementId = "DEF456";
+
+        [SetUp]
+        public void Arrange()
+        {
+            _orchestrator = new Mock<OrganisationOrchestrator>();
+            _owinWrapper = new Mock<IAuthenticationService>();
+            _featureToggle = new Mock<IAuthorizationService>();
+            _userViewTestingService = new Mock<IMultiVariantTestingService>();
+            _mapper = new Mock<IMapper>();
+            _flashMessage = new Mock<ICookieStorageService<FlashMessageViewModel>>();
+
+            _orchestrator.Setup(x => x.CreateLegalEntity(It.IsAny<CreateNewLegalEntityViewModel>()))
+                .Throws<UnauthorizedAccessException>()
+                //.ReturnsAsync(new OrchestratorResponse<EmployerAgreementViewModel>
```

Comment:
Does this commented out code need to be removed?

Reply:
yep, oversight. Good catch!